docker/README.md

```diff
@@ -0,0 +1,24 @@
+# Docker Image Tutorial
+Short tutorial on getting your Docker Image up and running. We will be assuming that you're using a SQLite database. Link to [Docker Installation](https://docs.docker.com/get-docker/)
+
+## Building the Image
+Manual build
+> docker build -t &lt;Image Name&gt; -f $PWD/Dockerfile .
+
+Pulling the Image from Docker Hub
+> docker pull rtohaan/alexandrie:latest
+
+## Changing the Port
```

Comment:
We can leave out instructions for changing the port and just reference the default config file (see my comment above)